carbon/components/cards/ChartCard/styles.module.scss

```diff
@@ -38,7 +38,11 @@
 }
 
 .cardHeaderTitle {
-  white-space: nowrap;
+  min-width: 30rem;
```

Comment:
What does the min width do in this case?

Reply:
I'm asking because it seems like it breaks the layout on small devices (iphone SE 2nd gen)

Reply:
It was to better use the available space on top. Let me try something better.